.devcontainer/devcontainer.json

```diff
@@ -5,7 +5,7 @@
   "postCreateCommand": "script/setup",
   "postStartCommand": "script/bootstrap",
   "containerEnv": { "DEVCONTAINER": "1" },
-  "appPort": ["8123:8123"],
+  "appPort": ["8123:8123", "5683:5683/udp"],
```

Comment:
Please add a comment that the port is used for shelly coap

Reply:
Done